src/Avalonia.Controls/TextBox.cs

```diff
@@ -97,6 +97,7 @@ public UndoRedoState(string text, int caretPosition)
         private UndoRedoHelper<UndoRedoState> _undoRedoHelper;
         private bool _ignoreTextChanges;
         private IEnumerable<Exception> _dataValidationErrors;
+        private string[] invalidCharacters = new String[1]{""};
```

Comment:
This looks like an empty string in the VS editor. Could you change it to read "\u007f" so it's more clear what's happening?

Also this doesn't need to be an instance member - it should be `static readonly`.